cfgov/unprocessed/js/config/error-messages-config.js

```diff
@@ -7,13 +7,21 @@
 'use strict';
 
 var ERROR_MESSAGES = {
-  CHECKBOX_ERRORS: {
-    required: 'Please select at least one of the "%s" options.'
+  CHECKBOX : {
+    required : 'Please select at least %s of the options.'
```

Comment:
Actually I see where this is used:

`ERROR_MESSAGES.CHECKBOX.required.replace( '%s', required );`

`replace` returns a new string, so `ERROR_MESSAGES.CHECKBOX.required` is constant because it's not modified directly. I'd say make it uppercase as well.